### PR TITLE
Revert bgu experiments and disable geordi logging

### PR DESCRIPTION
**Pull request by camallen**
~**DO NOT MERGE** until after the latest experiment is finished and the disabled geordi logging is tested.~

Revert the BGU experiment setup and ~disable~ remove Geordi logging everywhere. I've staged this code to https://www.galaxyzoo.org/beta for review before merging.

### CODE REVIEW

**Comment by camallen** [comment]
@mrniaboc are we good to shut this down now?

**Comment by astopy** [comment]
@camallen @mrniaboc Is this still relevant?

**Comment by camallen** [comment]
Yeah - the intervention is apparently still running and logging to geordi. @mrniaboc can we finally revert the experiment?

**Comment by mrniaboc** [comment]
The experiment is over for now, so this can be shut down.

File: app/lib/analytics.coffee [hunk 1]
```diff
@@ -68,21 +64,23 @@ logToGoogle = (eventData) =>
 This will log a user interaction both in the Geordi analytics API and in Google Analytics.
 ###
 logEvent = (params) =>
-  eventData = buildEventData(params)
-  addUserDetailsToEventData(eventData)
-  .always (eventData) =>
-    logToGeordi eventData
-    logToGoogle eventData
+  return true
```
Comment (astopy) [inline app/lib/analytics.coffee]: Should we just remove this function rather than commenting it out?

File: app/lib/analytics.coffee [hunk 2]
```diff
@@ -68,21 +64,23 @@ logToGoogle = (eventData) =>
...
 ###
 This will log an error in Geordi only.
 ###
 logError = (params) ->
-  eventData = buildEventData(params)
-  eventData['errorCode'] = params.errorCode
-  eventData['type'] = "error"
-  eventData['errorDescription'] = params.errorDescription
-  logToGeordi eventData
+  return true
```
Comment (astopy) [inline app/lib/analytics.coffee]: As above.

File: app/lib/analytics.coffee (hunk 1 above)
Comment (camallen) [reply app/lib/analytics.coffee]: yeah - i don't think we'll run the experiment again so i'll remove all the code related to event logging, etc.

**Pull request merged**